crates/theme_importer/Cargo.toml

```diff
@@ -23,5 +23,5 @@ serde.workspace = true
 serde_json.workspace = true
 simplelog = "0.9"
 strum = { version = "0.25.0", features = ["derive"] }
-theme = { path = "../theme", features = ["importing-themes"] }
```

Comment:
The use of the `importing-themes` feature here is intended, so we should keep it.

It's used to signify that the `theme_importer` is importing themes.

Reply:
Got it, I've added it back